### PR TITLE
fix(elevation): remove containment

### PR DESCRIPTION
**Pull request by rachelbt**
`contain: content` causes nested components to be un-shown because popup position is absolute.
see codepen: https://codepen.io/rachelbt/pen/jOGKqox

added ::slotted(*) for border-radius inherit (inherit doesn't)

### CODE REVIEW

**Comment by rachelbt** [comment]
I think its better that there will be a case of missing border-radius rather then hidden content

**Review by yinonov — COMMENTED** [review]
maybe we can remove border-radius on elevation altogether?

**Comment by rachelbt** [comment]
> 

It does :)

File: components/elevation/src/vwc-elevation.scss [hunk 1]
```diff
@@ -28,5 +28,8 @@ $vvd-elevation-border-radius: --vvd-elevation-border-radius;
 .vwc-elevation {
 	width: fit-content;
 	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
-	contain: content;
+}
+
+::slotted(*) {
+	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
 }
```
Comment (yinonov) [inline components/elevation/src/vwc-elevation.scss:35]: what if it's just an icon or text that will be assigned to slot?
this is unexpected as we have no control over the content.

need to brain storm again

**Review by yinonov — CHANGES_REQUESTED** [review]
I think we can just remove the border radius from elevation as drop-shadow will take into consideration the shape of content.
So we leave shaping responsibility to

File: components/elevation/src/vwc-elevation.scss [hunk 2]
```diff
@@ -28,5 +28,8 @@ $vvd-elevation-border-radius: --vvd-elevation-border-radius;
 .vwc-elevation {
 	width: fit-content;
 	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
-	contain: content;
+}
+
+::slotted(*) {
+	border-radius: var(#{$vvd-elevation-border-radius}, 6px);
```
Comment (yinonov) [inline components/elevation/src/vwc-elevation.scss:34]:
```suggestion
```

**Review by yinonov — APPROVED** [review]

**Pull request merged**